improver_tests/developer_tools/test_MOMetadataInterpreter.py

```diff
@@ -127,6 +128,20 @@ def test_handles_duplicate_model_string(probability_above_cube, interpreter):
     assert interpreter.model == "UKV"
 
 
+def test_vicinity_cell_method(
+    probability_over_time_in_vicinity_above_cube, interpreter
+):
+    """Test when precipitation accumulation in-vicinity cube has a cell method"""
+    cube = probability_over_time_in_vicinity_above_cube.copy()
+    diagnostic_name = "lwe_thickness_of_precipitation_amount"
+    cube.rename(f"probability_of_{diagnostic_name}_in_vicinity_above_threshold")
+    cube.cell_methods = []
+    cube.add_cell_method(
+        CellMethod(method="sum", coords="time", comments=(f"of {diagnostic_name}",),)
+    )
+    interpreter.run(cube)
```

Comment:
Is it correct that there are no assertions here? Something like `assert interpreter.methods == " sum over time"` could be considered?

Reply:
Kat queried this too. The point of the test is to make sure that the cube is now metadata compliant, which is true if the interpreter raises no exceptions. Double-checking the surrounding tests does suggest that an assertion like you suggest would be appropriate, so I've added one.